docs/cookbook/platform-sh-advanced.rst

```diff
@@ -0,0 +1,194 @@
+Advanced Platform.sh configurations
+===================================
+
+The basic set-up let's you easily set-up a Platform.sh project running your Sylius project. It should give you an
```

Comment:
Shouldn't we try to avoid repetition of the same words in one sentence? `Platform.sh platform` / `Platform.sh deployment server` / `Platform.sh PaaS`?

Reply:
Sure. Makes sense to come up with one single name and use that one everywhere :-)